modules/rostests/apitests/ntdll/RtlMultipleAllocateHeap.c

```diff
@@ -0,0 +1,300 @@
+/*
+ * PROJECT:         ReactOS API tests
+ * LICENSE:         LGPLv2.1+ - See COPYING.LIB in the top level directory
+ * PURPOSE:         Test for RtlMultipleAllocateHeap and RtlMultipleFreeHeap
+ * PROGRAMMER:      Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
+ */
+
+#include <apitest.h>
+#include <windows.h>
```

Comment:
Windows include in ntdll apitest seems weird?

Reply:
Used `"precomp.h"`.